databases/week3/sql_week3.sql

```diff
@@ -0,0 +1,237 @@
+CREATE DATABASE meal_sharing;
+USE meal_sharing;
+
+CREATE TABLE meal(
+ id INT UNSIGNED PRIMARY KEY AUTO_INCREMENT,
+ title VARCHAR (255),
+ description TEXT,
+ location VARCHAR(255) NOT NULL,
+ event_day DATETIME NOT NULL,
+ max_reservations INT NOT NULL,
+ price DECIMAL,
```

Comment:
It's best to always specify the amount of digits for decimal. Something like `DECIMAL(10,2)` would work. Also probably best to not allow null

Reply:
I had later realised and added the decimals using alter table. But yes I will drop the table and make the changes again as you advised below